tests/validation/cattlevalidationtest/core/test_host_api.py

```diff
@@ -0,0 +1,39 @@
+from common_fixtures import *  # NOQA
+from test_docker import docker_context
+import websocket as ws
+# work around flake8 issue
+docker_context
```

Comment:
I just followed the pattern from existing test_host_stats.py file here to use docker_context - https://github.com/rancherio/cattle/blob/master/tests/integration/cattletest/core/test_host_stats.py